src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -214,15 +218,29 @@ public void postToldpcvOfLdprWithMementoDatetimeShouldCreateNewResource(final St
                                         "state given in the request body.",
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-post",
                                         ps);
-
+        //create a versioned resource
+        final Response createResponse = createVersionedResource(uri, info);
+        //if post is supported on the ldpcv
+        // get the new resource, which should have all the proper memento headers
+        final String originalResource = getLocation(createResponse);
+        final Response originalResponse = doGet(originalResource);
+        final String body = originalResponse.getBody().asString();
+        final URI timeMapURI = getTimeMapUri(originalResponse);
+        if (hasHeaderValueInMultiValueHeader("Allow", "POST", doGet(timeMapURI.toString()))) {
+            //create a memento using the Memento-Datetime and a body
+            final String mementoUri = createMemento(originalResource, "Sat, 1 Jan 2000 00:00:00 GMT", body);
+            // is the memento exactly the same as the original?
+            confirmResponseBodyNTriplesAreEqual(originalResponse, doGet(mementoUri));
```

Comment:
This test should be verifying that the content (body) of the memento matches what was provided as the body when creating the memento. However, since the provided body is the same as the current state of the LDPRv, it is unclear if the provided body was used in creating the memento or the current state.
I would suggest pushing new content, not the content of the current state, as the memento body.